build.go

```diff
@@ -70,6 +76,26 @@ func Build(entryResolver EntryResolver, installProcess InstallProcess, siteProce
 			return packit.BuildResult{}, err
 		}
 
+		logger.GeneratingSBOM(packagesLayer.Path)
+
+		var sbomContent sbom.SBOM
+		duration, err = clock.Measure(func() error {
+			sbomContent, err = sbomGenerator.Generate(context.WorkingDir)
```

Comment:
Why generate on the working directory instead of `packagesLayer` if that's the path we're logging about?